DOCSP-44700-pre-split-chunks (#452)
mongodb/docs-cluster-to-cluster-sync: source/reference/api/start.txt

Pre-Split Chunks
~~~~~~~~~~~~~~~~

When ``mongosync`` syncs to a sharded destination cluster, it pre-splits chunks 
for sharded collections on the destination cluster. For each sharded collection, 
``mongosync`` creates twice as many chunks as there are shards in the 

> destination cluster. This is supported in the following configurations:

Reviewer: small suggestion for active voice (just spitballing, not sure if "procedure" is the right word here)

```suggestion
destination cluster. The following configurations support this procedure:
```

Author: Makes sense! Updating.

- Sync from a replica set to a sharded cluster.

> - Sync between sharded clusters.

Reviewer 2: I'd say that these bullets are de facto covered by us saying "When `mongosync` syncs to a sharded destination cluster" and that we can remove them, but I'll leave the final decision up to you.

Author: Ah gotcha ! I'll opt for removing then :)
